src/olympia/addons/buttons.py

```diff
@@ -12,7 +12,8 @@
 @jinja2.contextfunction
 def install_button(context, addon, version=None, show_contrib=True,
                    show_warning=True, src='', collection=None, size='',
-                   detailed=False, impala=False, latest_beta=False):
+                   detailed=False, impala=False, latest_beta=False,
+                   show_download_anyway=False):
```

Comment:
I wonder, is there anywhere we _don't_ do `show_download_anyway` now?

Reply:
Yes. The homepage/listing pages, and also for the "related" add-ons on the detail page (like the "add-ons from this developer" shown on an add-on detail page)